packages/data-context/src/actions/WizardActions.ts

```diff
@@ -148,6 +161,20 @@ export class WizardActions {
     }
   }
 
+  async detectFrameworks () {
+    if (!this.ctx.currentProject) {
+      return
+    }
+
+    const officialFrameworks = CT_FRAMEWORKS.map((framework) => resolveComponentFrameworkDefinition(framework))
+    const thirdParty = await detectThirdPartyCTFrameworks(this.ctx.currentProject)
```

Comment:
Found a race condition, if this is slow, the UI is wrong

<img width="1329" alt="image" src="https://user-images.githubusercontent.com/19196536/218369272-941c96d6-ee38-43c8-b5ff-055502fc739f.png">
https://app.circleci.com/pipelines/github/cypress-io/cypress/49442/workflows/64f996ae-9b7e-4e74-b7d7-a091da0047c6/jobs/2054201/tests#failed-test-0

I will fix it